stdlib/Std/src/Base.luna

```diff
@@ -842,10 +853,10 @@ class Map k v:
             False: case (k < ky) of
                 True:
                     newL = l.insert k v
-                    Map._bin ky y newL r . _balanceL
+                    Map._bin ky y newL r
```

Comment:
Are we not balancing anymore?

Reply:
Yeeah :( The TC performance with this gets so bad that I find myself disabling this all the time.